### PR TITLE
Right! PRs!

### PR DESCRIPTION
**Pull request by workingjubilee**
dot dot dot!

### CODE REVIEW

File: src/06_tuples.py [hunk 1]
```diff
@@ -17,24 +17,31 @@
...
 # Write a function `print_tuple` that prints all the values in a tuple
 
-# YOUR CODE HERE
+def print_tuple(t):
+    try:
+        for i in t:
+            print(i)
+    except:
+        print(t)
+
```
Comment (Addison-Stavlo) [inline src/06_tuples.py:44]: I really like the use of try/except

File: src/15_classes.py [hunk 2]
```diff
@@ -1,29 +1,57 @@
...
 
 # Without changing the following line, how can you make it print into something
 # more human-readable? Hint: Look up the `object.__str__` method
 print(waypoint)
 
 # Make a new geocache "Newberry Views", diff 1.5, size 2, 44.052137, -121.41556
 
-# YOUR CODE HERE
+geocache = Geocache(name="Newberry Views", difficulty=1.5,
+                    size=2, lat=44.052137, lon=-121.41556)
```
Comment (Addison-Stavlo) [inline src/15_classes.py:54]: I assume you already know this, but if you put the values in order as assigned in the constructor you can get rid of all the variable names in the function call... 
ex) Geocache(44.05, -121.42, "Newberry Views", 1.5, 2)
i wouldn't say either way is better than the other, they both have their trade offs for readability and all that.

**Review by Addison-Stavlo — APPROVED** [review]
Great work!  I definitely believe you have used python before, lol!

File: src/15_classes.py (hunk 2 above)
Comment (workingjubilee) [reply src/15_classes.py:54]: True. I felt like I kind of had to do it with lat/lon and by then when I was doing that, it was downhill from there... I thought to myself "I might as well specify everything, right?"

**Pull request merged**